Packs/1Password/Integrations/OnePasswordEventCollector/OnePasswordEventCollector.py

```diff
@@ -0,0 +1,392 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from CommonServerUserPython import *  # noqa
+
+import urllib3
+from http import HTTPStatus
+from requests.structures import CaseInsensitiveDict
+from datetime import datetime, timedelta
+
+# Disable insecure warnings
+urllib3.disable_warnings()
```

Comment:
I think we can omit this, since it runs in CSP

Reply:
Removed this line (and the related import)